code/game/objects/structures/signs.dm

```diff
@@ -148,4 +148,48 @@
 /obj/structure/sign/directions/evac
 	name = "escape arm"
 	desc = "A direction sign, pointing out which way the escape shuttle dock is."
-	icon_state = "direction_evac"
\ No newline at end of file
+	icon_state = "direction_evac"
+
+/obj/structure/sign/safetyboard //64 by 32 pixels, takes two tiles
+	name = "Station Safety Sign"
+	desc = "Only you can prevent station spaghettification."
+	icon = 'icons/obj/safetyboards.dmi'
+	icon_state = "major_accident"
+	var/value
+	var/record
+	var/record_desc = " The gold trim signifies that we've reached the current record for safety; no pressure!"
+
+/obj/structure/sign/safetyboard/New()
+	while(ticker.current_state < GAME_STATE_PLAYING)
+		sleep(10)
+	if(ticker.persist && !value)
+		value 	= ticker.persist.last_singulo_release
+		record 	= ticker.persist.record_singulo_release
+
+	var/icon/I
+	if(value == 0)
+		I = icon('icons/obj/safetyboards.dmi',"X0-red")
+		overlays += I
+		return
+	else if(value < 10)
+		I = icon('icons/obj/safetyboards.dmi',"X[value]")
+		overlays += I
+	else
+		var/ones = value - round(value*0.1)*10 //round(X) is floor(), round(X,1) is round [but not round()]
+		I = icon('icons/obj/safetyboards.dmi',"X[ones]")
+		overlays += I
+	if(value > 9)
+		var/tens =  round(value/10)
+		I = icon('icons/obj/safetyboards.dmi',"[tens]X")
+		overlays += I
+	if(value == record)
+		I = icon('icons/obj/safetyboards.dmi',"record")
+		overlays += I
+		desc += record_desc
+	if(value > 99) //I'll eat my hat

```

Comment:
watch as basil/bagil/badger/sybil 2/insert politcally correct name that I will get murdered if I don't use for the second server reaches this